cli/exp_scaletest.go

```diff
@@ -607,7 +602,7 @@ func (r *RootCmd) scaletestCreateWorkspaces() *clibase.Cmd {
 			richParameters, err := prepWorkspaceBuild(inv, client, prepWorkspaceBuildArgs{
 				Action:           WorkspaceCreate,
 				Template:         tpl,
-				NewWorkspaceName: "scaletest-%", // TODO: the scaletest runner will pass in a different name here. Does this matter?
+				NewWorkspaceName: "scaletest-N", // TODO: the scaletest runner will pass in a different name here. Does this matter?
```

Comment:
I presume that the comment is still valid?

Reply:
As we saw, it does appear to matter if `NewWorkspaceName` ends up not being valid :-)